about.js

```diff
@@ -0,0 +1,19 @@
+window.addEventListener('DOMContentLoaded', () => {
+  const wrap = document.getElementById('nav-wrapper');
+  const openWrap = document.getElementById('wrapper');
+  const closeWrap = document.getElementById('close-wrap');
+  function wrapFunction() {
+    if (wrap.style.display === 'flex') {
+      wrap.style.display = 'none';
+    } else {
+      wrap.style.display = 'flex';
+    }
+  }
+  openWrap.addEventListener('click', wrapFunction);
+  closeWrap.addEventListener('click', wrapFunction);
+  const imgHomeSetup = document.querySelectorAll('.img-home-setup');
+  function homeFunction() {
+    window.location.href = 'index.html';
+  }
+  imgHomeSetup.forEach((img) => { img.addEventListener('click', homeFunction); });
+});
```

Comment:
- [ ] I noticed that you have exactly the same code in `index.js` file, So I suggest deleting this file and you can use `  <script src="index.js"></script>` in your `about.html` page instead 💡

> To follow [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) and to Keep your [code clean](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices) 💯